apps/docs/content/guides/self-hosting/enable-mcp.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: 'Enabling MCP Server Access'
+description: 'Configure secure access to the MCP server in your self-hosted Supabase instance.'
+subtitle: 'Configure secure access to the MCP server in your self-hosted Supabase instance.'
+---
+
+The MCP (Model Context Protocol) server in [self-hosted Supabase](/docs/guides/self-hosting/docker) runs behind the Studio API. Currently, it does not offer OAuth 2.1 authentication, and is not intended to be exposed to the Internet. The corresponding API route has to be protected by restricting network connections from the outside. By default, all connections to the MCP server are denied.
+
+This guide explains how to securely enable access to your self-hosted MCP server.
+
+## Security considerations
+
+<Admonition type="caution">
+
+Do not allow connections to the self-hosted MCP server from the Internet. Only access it via:
+
+- A VPN connection to the server running the Studio container
+- An SSH tunnel from your local machine
+
+</Admonition>
+
+## Accessing via SSH tunnel
+
+### Step 1: Determine the local IP address that will be used to access the MCP server
+
+When connecting via an SSH tunnel to the Studio Docker container, the source IP will be that of the Docker bridge gateway. You need to allow connections from this IP address.
+
+Determine the Docker bridge gateway IP on the host running your Supabase containers:
+
+```bash
+docker inspect supabase-kong \
+  --format '{{range .NetworkSettings.Networks}}{{println .Gateway}}{{end}}'
+```
+
+This command will output a private range IP address, e.g., `172.18.0.1`.
```

Comment:
"private range" an important detail?

can it just be:  This command will output an IP address like `172.18.0.1`.